packages/webpack/src/builder.js

```diff
@@ -57,6 +57,12 @@ export class WebpackBundler {
       webpackConfigs.push(this.getWebpackConfig('Server'))
     }
 
+    const additionalConfigs = await this.buildContext.nuxt.callHook('build:config')
+
+    if (additionalConfigs) {
+      webpackConfigs.push(...[].concat(additionalConfigs))
```

Comment:
Why the `[].concat`? If the array contains objects then after the concat its still an array to the same references right?

Reply:
The concat here is for supporting return value either array or object as single or multiple builds

Reply:
Thanks, actually I didnt know you could return values with [`callHook`](https://github.com/jsless/hable/blob/master/src/hable.js#L45). Maybe I am just overseeing something simple but there is no return statement or whatever, so how does it even return a value?

Reply:
Yes, that's true, it has been changed for a while, I've changed in new pr